pkg/strategy/dca2/maker.go

```diff
@@ -105,3 +109,24 @@ func (s *Strategy) generateMakerOrder(short bool, budget, price, margin fixedpoi
 
 	return submitOrders, nil
 }
+
+func (s *Strategy) cancelMakerOrders(ctx context.Context) error {
+	s.logger.Info("[DCA] cancel maker orders")
+	e, ok := s.Session.Exchange.(cancelOrdersByGroupIDApi)
+	if ok {
+		cancelledOrders, err := e.CancelOrdersByGroupID(ctx, int64(s.OrderGroupID))
+		if err != nil {
+			return err
+		}
+
+		for _, cancelledOrder := range cancelledOrders {
+			s.logger.Info("CANCEL ", cancelledOrder.String())
+		}
+	} else {
+		if err := s.OrderExecutor.ActiveMakerOrders().GracefulCancel(ctx, s.Session.Exchange); err != nil {
```

Comment:
can be just s.OrderExecutor.GracefulCancel

Reply:
if we use GracefulCancel, we still need to GracefulCancel(ctx, s.OrderExecutor.ActiveMakerOrders().Orders()...)
I think it's almost the same ?

Reply:
yes